js/src/controls/SearchControl.js

```diff
@@ -10,13 +10,13 @@ export class LeafletSearchControlModel extends control.LeafletControlModel {
       _view_name: 'LeafletSearchControlView',
       _model_name: 'LeafletSearchControlModel',
       url: null,
-      jsonpParam: 'json_callback',
-      propertyName: 'display_name',
-      propertyLoc: ['lat','lon'],
-      autoType: false,
-      autoCollapse: false,
+      jsonp_param: 'json_callback',
+      property_name: 'display_name',
+      property_loc: ['lat','lon'],
+      auto_type: false,
+      auto_collapse: false,
       zoom:10,
-      animateLocation:false,
+      animate_location:false,
       marker: L.circleMarker([0,0],{radius:30}),
```

Comment:
This should be `null` by default.

You also need to specify a special widget serializer/deserializer for the `marker` attribute. You can look at the `Popup` implementation for an example: https://github.com/jupyter-widgets/ipyleaflet/blob/master/js/src/layers/Popup.js#L29.

Reply:
Done.